utils/create_srg_export.py

```diff
@@ -220,6 +444,17 @@ def get_rule_json(json_path: str) -> dict:
     return rule_json
 
 
+def add_variables(env_yaml, policy):
+    for control in policy.controls:
+        for rule in control.selections:
+            if rule.startswith('var_'):
+                parts = rule.split('=')
+                if len(parts) != 2:
+                    raise ValueError('Invalid variable definition')
+                env_yaml[parts[0].replace('var_', '')] = parts[1]
```

Comment:
Please use an explanatory variable here. And also double-check whether removal of `var_` is the right thing to do - why we want to do it in the first place?